test/test_cve_bin_tool.py

```diff
@@ -0,0 +1,157 @@
+        "CVE-2018-10916": {"fixed_release": "4.8.4-1"},
+        "CVE-2021-23177": {"fixed_release": "3.4.3-2+deb11u1"},
+        "CVE-2021-31566": {"fixed_release": "3.4.3-2+deb11u1"},
+        "CVE-2010-4252": {"fixed_release": "0"},
+        "CVE-2010-5298": {"fixed_release": "1.0.1g-3"},
+        "CVE-2011-1945": {"fixed_release": "1.0.0e-1"},
+        "CVE-2011-4108": {"fixed_release": "1.0.0f-1"},
+        "CVE-2011-4576": {"fixed_release": "1.0.0f-1"},
+        "CVE-2011-4577": {"fixed_release": "1.0.0f-1"},
+        "CVE-2011-4619": {"fixed_release": "1.0.0h-1"},
+        "CVE-2012-0027": {"fixed_release": "1.0.0f-1"},
+        "CVE-2013-6449": {"fixed_release": "1.0.1e-5"},
+        "CVE-2014-0076": {"fixed_release": "1.0.1g-1"},
+        "CVE-2015-4000": {"fixed_release": "1.0.2b-1"},
+        "CVE-2016-2106": {"fixed_release": "1.0.2h-1"},
+        "CVE-2016-2107": {"fixed_release": "1.0.2h-1"},
+        "CVE-2016-2108": {"fixed_release": "1.0.2c-1"},
+        "CVE-2016-2109": {"fixed_release": "1.0.2h-1"},
+        "CVE-2016-2176": {"fixed_release": "0"},
+        "CVE-2016-7056": {"fixed_release": "1.0.2a-1"},
+        "CVE-2021-4044": {"fixed_release": "0"},
+        "CVE-2010-2477": {"fixed_release": "1.7.4-1"},
+    }
+
+    def test_update_db(self, mocker: MockerFixture):
+        mocker.patch("subprocess.run")
+        spy_subprocess = mocker.spy(subprocess, "run")
+        cve_bin_tool_scanner = cve_bin_tool.CVE_BIN_TOOL()
+        cve_bin_tool_scanner.update_db("NVD_API_KEY")
+        spy_subprocess.assert_called_once_with(
+            [
+                "cve-bin-tool",
+                "--use-mirror",
+                "https://raw.githubusercontent.com/sec-data/mirror-sandbox/main",
+                "--verify",
+                str((Path(".").parent / "assets" / "mirror" / "pub.asc").absolute()),
+                "--nvd-api-key",
+                "NVD_API_KEY",
+            ]
+        )
+        mocker.stopall()
+
+    def test_scanner(self, mocker: MockerFixture):
+        mocker.patch("subprocess.run", self.__return_console_output)
+        spy_subprocess = mocker.spy(subprocess, "run")
+        cve_bin_tool_scanner = cve_bin_tool.CVE_BIN_TOOL()
+        available_fixes = cve_bin_tool_scanner.scan(
+            ".",
+            formats=["html", "pdf", "json"],
+            output="report",
+            exclude="test",
+        )
+        spy_subprocess.assert_called_once_with(
+            [
+                "cve-bin-tool",
+                ".",
+                "--detailed",
+                "--available-fix",
+                "ubuntu-hirsute",
+                "--format",
+                "html,pdf,json",
+                "--output-file",
+                "report",
+                "--exclude",
+                "test",
+            ],
+            capture_output=True,
+            text=True,
+        )
+        assert available_fixes == self.AVAILABLE_FIXES
```

Comment:
This is likely to be a bit brittle as issues change over time, as we've seen with similar tests in the main tool.  I'd suggest for future proofing that we change it to something more like...

```suggestion
        assert self.AVAILABLE_FIXES.issubset(available_fixes)
```